CRNotifications/Classes/CRNotification.swift

```diff
@@ -39,7 +39,7 @@ public class CRNotificationView: UIView, CRNotification {
 	}()
 	
     private var completion: () -> () = {}
-    
+    public var onClickDelegate: CRNotificationDelegate?
```

Comment:
Maybe this could just be called `delegate`? Then other behaviour might be added in the future, such as a swipe, manual hiding or even buttons below the text. As such we are no longer limited by the name. I was thinking to add two small reaction buttons at the bottom. Then that button can be forwarded here, too 🙂

Reply:
yes, calling it delegate sounds good!